### PR TITLE
Tidy up entry

### LINKED ISSUE #32: Entry point script
Moving this to its own issue, as we currently have a different entry point script in each branch, soon all to be merged into master.

Currently 
- `develop` runs kaska in each tile and then combines them (in parallel using dask if available)
- `sar_wcm` runs kaska_sar on the whole image
- `entry_point` runs kaska on the whole image, but contains the right way to define a script in a python package

### LINKED ISSUE #33: Refactor logger setup from various modules/files
Probably into its own function in utils.

### LINKED ISSUE #63: Expose dask to entry point script/function

### PR DESCRIPTION
**Pull request by ilectra**
Removed un-needed, duplicated entry functions and scripts. In the process, some tidy up of other open issues:
Fixes #33 
Fixes #32 
Fixes #63  (Changes ported over from #65 )

### CODE REVIEW

File: scripts/run_kaska [hunk 1]
```diff
@@ -1,33 +1,69 @@
...
 args = arg_parser.parse_args()
-#print(args.start_date, args.end_date, args.temporal_grid_space, args.parent_folder, args.state_mask)
+#print(args.start_date, args.end_date, args.temporal_grid_space, args.parent_folder, args.state_mask, args.output_folder, args.loglevel, args.logfile, args.block_size)
+
+# Setup logger
+loglevels = { 'debug':logging.DEBUG,
+              'info':logging.INFO,
+              'warning':logging.WARNING,
+              'error':logging.ERROR }
+LOG = logging.getLogger("KaSKA")
```
Comment (jgomezdans) [inline scripts/run_kaska]: you get use `get_logger` for this

File: scripts/run_kaska [hunk 2]
```diff
@@ -1,33 +1,69 @@
...
 #state_mask = "/home/ucfajlg/Data/python/KaFKA_Validation/LMU/carto/ESU.tif"
 
-run_process(args.start_date, args.end_date, args.temporal_grid_space,
-             args.parent_folder, args.state_mask, args.output_folder)
+# Prepare arguments needed to run main kaska executable
+s2_emulator = get_emulator("prosail", "Sentinel2")
+approx_inverter = get_inverter("prosail_5paras", "Sentinel2")
+dask_client = None
+
+kaska_runner(args.start_date, args.end_date, args.temporal_grid_space, args.state_mask,
```
Comment (jgomezdans) [inline scripts/run_kaska]: On mobile, but I assume that you can just import kaska and call kaska_runner directly right?

File: scripts/run_kaska [hunk 3]
```diff
@@ -1,33 +1,69 @@
...
+LOG.propagate = False
+LOG.info("Running KaSKA...")
 
 #start_date = dt.datetime(2017, 5, 1)
 #end_date = dt.datetime(2017, 9, 1)
 #temporal_grid_space = 5
 #parent_folder = "/home/ucfajlg/Data/python/KaFKA_Validation/LMU/s2_obs/"
 #state_mask = "/home/ucfajlg/Data/python/KaFKA_Validation/LMU/carto/ESU.tif"
 
-run_process(args.start_date, args.end_date, args.temporal_grid_space,
```
Comment (jgomezdans) [inline scripts/run_kaska]: I think you still want to have a `run_process` function exposed to the user (e.g. available via `import kaska`)

File: scripts/run_kaska (hunk 1 above)
Comment (ilectra) [reply scripts/run_kaska]: Yes, much better this way, thanks!

File: scripts/run_kaska (hunk 2 above)
Comment (ilectra) [reply scripts/run_kaska]: Yes, you can. If anything this script is an example of exactly how to do that in your own script.
Having the CLI doesn't prevent anyone from using kaska as a library, as intended.

File: scripts/run_kaska (hunk 3 above)
Comment (ilectra) [reply scripts/run_kaska]: ok, you're right, I added it back, and will document it as well. This function should now be called by _ALL_ entry scripts, no more `__main__`'s.

**Comment by ilectra** [comment]
@jgomezdans , I think I addressed your comments, please approve and merge if you're happy! Remember I implemented some of your changes from #65 as well.

**Pull request merged**